ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/vr/RDXVRModeManager.java

```diff
@@ -264,13 +264,13 @@ private void getRenderables(Array<Renderable> renderables, Pool<Renderable> pool
             }
             case JOYSTICK_WALKING -> joystickBasedStepping.getRenderables(renderables, pool);
          }
-      }
 
-      if (stereoVision.isEnabled())
-         stereoVision.getDualBlackflySphericalProjection().getRenderables(renderables, pool, sceneLevels);
+         if (stereoVision.isEnabled())
+            stereoVision.getDualBlackflySphericalProjection().getRenderables(renderables, pool, sceneLevels);
 
-      if (vrModeControls.getRenderOnLeftHand().get())
-         vrModeControls3DPanel.getRenderables(renderables, pool);
+         if (vrModeControls.getRenderOnLeftHand().get())
+            vrModeControls3DPanel.getRenderables(renderables, pool);
+      }
    }
```

Comment:
Moving this bracket fixes and unrelated bug where these renderables would be seen by the point cloud -- they should be in the virtual scene level.